### PR TITLE
docs: contributing guide - prettier - small PRs

### PR DESCRIPTION
**Pull request by alex-okrushko**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- Add instructions to run `prettier` on live examples
- Add a note about potential Design Doc and smaller PRs

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/content/marketing/contributing.md [hunk 1]
```diff
@@ -50,6 +50,20 @@ yarn serve-and-sync
 
 After running this command, an instance of the website and docs will be available at `http://localhost:4200`
 
+### Adding live examples
+
+Live examples is the best way to provide code snippets in the guide, for the following reasons:
+* it's easier to maintain them
+* they prove that the code example is working
+* a developer can open the full example in the StackBlitz to follow along
+* the formatting of all of the example is kept consistent
```
Comment (timdeschryver) [inline projects/ngrx.io/content/marketing/contributing.md:59]:
```suggestion
* the formatting of all of the examples is kept consistent

```

File: projects/ngrx.io/content/marketing/contributing.md [hunk 2]
```diff
@@ -50,6 +50,20 @@ yarn serve-and-sync
...
 
+### Adding live examples
+
+Live examples is the best way to provide code snippets in the guide, for the following reasons:
+* it's easier to maintain them
+* they prove that the code example is working
+* a developer can open the full example in the StackBlitz to follow along
+* the formatting of all of the example is kept consistent
+
+To keep the formatting consistent, run the following command after any of the live examples are added or modified:
```
Comment (timdeschryver) [inline projects/ngrx.io/content/marketing/contributing.md:61]: Don't we have a git-hook for this?

File: projects/ngrx.io/content/marketing/contributing.md [hunk 3]
```diff
@@ -118,7 +132,9 @@ yarn test
 - We value keeping the API surface small and concise, which factors into whether new features are accepted.
 - Submit an issue with the prefix `RFC:` with your feature request.
 - The feature will be discussed and considered.
-- Once the PR is submitted, it will be reviewed and merged once approved.
+- If the feature is large in scope, the Design Doc might be requested.
```
Comment (timdeschryver) [inline projects/ngrx.io/content/marketing/contributing.md:135]: Can we link to a design doc template here?

**Review by timdeschryver — COMMENTED** [review]
👍

File: projects/ngrx.io/content/marketing/contributing.md (hunk 2 above)
Comment (alex-okrushko) [reply projects/ngrx.io/content/marketing/contributing.md:61]: Nope - this is new.

**Review by timdeschryver — APPROVED** [review]
Fyi, the git-hook isn't formatting the examples because these are excluded at https://github.com/ngrx/platform/blob/master/.prettierignore#L5
I think it would be better to reconfigure it, so it does format the examples.

**Comment by alex-okrushko** [comment]
> Fyi, the git-hook isn't formatting the examples because these are excluded at https://github.com/ngrx/platform/blob/master/.prettierignore#L5
> I think it would be better to reconfigure it, so it does format the examples.

Thanks Tim! I'll adjust that.

**Comment by brandonroberts** [comment]
The examples aren't formatted currently because it can impact the included snippets for regions of code

**Comment by alex-okrushko** [comment]
> The examples aren't formatted currently because it can impact the included snippets for regions of code

I was planning to adjust to something like:
```
/projects/ngrx.io
!/projects/ngrx.io/content/examples
```

**Pull request merged**